src/Form/FastlySettingsForm.php

```diff
@@ -258,14 +281,32 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['purge']['purge_actions'] = [
       '#type' => 'details',
       '#title' => $this->t('Purge actions'),
+      '#description' => $this->t('Purge / invalidate all site content: affects this site only. <br><b>WARNING: PURGE WHOLE SERVICE ACTION WILL DESTROY THE ENTIRE CACHE FOR ALL SITES IN THE CURRENT FASTLY SERVICE.</b>'),
```

Comment:
```suggestion
      '#description' => $this->t('Purge / invalidate site content.'),
```

Reply:
This is description for both actions (Purging site content and purging whole service)  not just site content one as this is a wrapper for both of them. If I replace this with your suggestion then this will reference only one of them.